outbox/src/main/kotlin/com/ritense/outbox/publisher/PollingPublisherJob.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.outbox.publisher
+
+import org.springframework.scheduling.annotation.Scheduled
+
+class PollingPublisherJob(
+    private val pollingPublisherService: PollingPublisherService
+) {
+
+    @Scheduled(cron = "\${valtimo.outbox.pollingpublisher.job:*/10 * * * * *}")
```

Comment:
I prefer using `fixedRateString` here, because it really a rate you'd want to specify here, and not 'once every sunday in an odd week in may'